Client/Application/Delegates/AppDelegate.swift

```diff
@@ -61,6 +60,28 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
         self.window!.backgroundColor = .black
         
         // Brave Core Initialization
+        BraveCoreMain.setLogHandler { severity, file, line, messageStartIndex, message in
```

Comment:
Does it get stripped in production builds?

Reply:
i mean, are Reward logs running in prod, i think no, but want to make sure,
maybe it's required for Rewards internals or something idk

Reply:
Rewards logs were always ran in prod, but not persisted. This is the same as before, since this method just forwards it to the XCGLogger responsible for logging. Previously it was just ads/ledger, now it is anything that is logged from Chromium side from a `brave/*` directory.

Reply:
alright, fine by me

Reply:
> Rewards logs were always ran in prod, but not persisted

Sorry that was incorrect, console destinations are removed in public builds: https://github.com/brave/brave-ios/blob/0b76a6a2018227761369616d21dfc0c2660491d5/Shared/Logger.swift#L30

Though this change uses the same logger as mentioned previously, so it will remain removed from public builds

Reply:
perfect, thanks for double checking